NetTally.Core/Output/TallyOutput.cs

```diff
@@ -78,6 +83,30 @@ private void BuildGlobal(CancellationToken token)
         }
 
         /// <summary>
+        /// Constructs output if one of the formatting options is chosen.
+        /// </summary>
+        /// <returns>true if output is constructed, false otherwise.</returns>
+        private bool ConstructFormattingOutput(CancellationToken token)

```

Comment:
This really ought to be incorporated into ConstructRankedOutput().  It doesn't handle multiple tasks, and uses code that's the same as the other function.  The only difference is that this bails out entirely rather than letting the remaining vote info be output, and there's really no reason to do that.